parse/file.go

```diff
@@ -35,6 +35,16 @@ func fileGoType(x ast.Expr) string {
 	}
 }
 
+func commentText(g *ast.CommentGroup) string {
+	// this code used to just call g.Text(), but the behavior of this method changed in Go 1.15:
+	// https://go-review.googlesource.com/c/go/+/224737
+	res := make([]string, len(g.List))
+	for i, c := range g.List {
+		res[i] = c.Text
+	}
+	return strings.Join(res, "\n")
```

Comment:
What if it's called from Windows? Does `\n` fit all the OS?

Reply:
It doesn't really matter

Reply:
ok!